changelog.md

```diff
@@ -1,3 +1,142 @@
+* driver: save performance.now() to avoid conflict ([#6387](https://github.com/googlechrome/lighthouse/pull/6387))
+* driver: request smaller trace in m71+ ([#6117](https://github.com/googlechrome/lighthouse/pull/6117))
+* driver: stringify protocol error messages ([#6302](https://github.com/googlechrome/lighthouse/pull/6302))
+* driver: address bckenny feedback on #6117 ([#6285](https://github.com/googlechrome/lighthouse/pull/6285))
+* driver: reduce required traceCategories again ([54baf91](https://github.com/googlechrome/lighthouse/commit/54baf91))
+* driver: recover from rejection on handleJavaScriptDialog ([#6327](https://github.com/googlechrome/lighthouse/pull/6327))
+* emulation: fix missing space in mobile UA ([#6161](https://github.com/googlechrome/lighthouse/pull/6161))
+* icons: defer to manifest-icon type hint for image type ([#6230](https://github.com/googlechrome/lighthouse/pull/6230))
+* network: handle invalid NetworkRequests ([#6157](https://github.com/googlechrome/lighthouse/pull/6157))
+* network-analyzer: move throughput to NetworkAnalyzer ([#5900](https://github.com/googlechrome/lighthouse/pull/5900))
+* perf: add timing instrumentation to measure runtime perf ([#3745](https://github.com/googlechrome/lighthouse/pull/3745))
+* proto: add proto definition for LHR ([#6183](https://github.com/googlechrome/lighthouse/pull/6183))
+* simulator: start nodes in observed order ([#5362](https://github.com/googlechrome/lighthouse/pull/5362))
+* unminified-javascript: replace esprima with custom tokenizer ([#5925](https://github.com/googlechrome/lighthouse/pull/5925))
+* remove recoverOrThrow / err.fatal ([#6343](https://github.com/googlechrome/lighthouse/pull/6343))
+* convert computed artifact loading to regular require() ([#6204](https://github.com/googlechrome/lighthouse/pull/6204))
+* remove no-websql audit ([#6293](https://github.com/googlechrome/lighthouse/pull/6293))
+* bail on run if insecure ssl certification ([#6300](https://github.com/googlechrome/lighthouse/pull/6300))
+* make throttled offscreen-images work without TTI ([#6039](https://github.com/googlechrome/lighthouse/pull/6039))
+* save HTMLElement.p.matches function to avoid conflict ([#6283](https://github.com/googlechrome/lighthouse/pull/6283))
+* remove fonts gatherer, move font-display to use CSSUsage artifact ([#6166](https://github.com/googlechrome/lighthouse/pull/6166))
+* remove last dependency on WebInspector ([#6209](https://github.com/googlechrome/lighthouse/pull/6209))
+* add timeout to all protocol commands ([#6347](https://github.com/googlechrome/lighthouse/pull/6347))
+* don't load blank page twice in first pass ([#6369](https://github.com/googlechrome/lighthouse/pull/6369))
+* refactor finding of trace start ([#6370](https://github.com/googlechrome/lighthouse/pull/6370))
+* increase Page.navigate timeout ([#6412](https://github.com/googlechrome/lighthouse/pull/6412))
+
```

Comment:
```suggestion
* add LH.Result object support to prepareLabData ([#6432](https://github.com/GoogleChrome/lighthouse/pull/6432))
```

Reply:
OK, I'll add this just before releasing (so I don't gotta keep merging master + updating the changelog)

Reply:
it's fine to leave this particular commit out of the alpha.